docs/sequence.puml

```diff
@@ -0,0 +1,58 @@
+@startuml
+database Bitcoin
+actor Bob
+Actor Alice
+database Monero
+
+Bob --> Alice: Establish connection
+
+group Request Response Channel
+    Bob -> Alice: Request Amounts
+    note left: Btc Amount
+
+    Alice --> Bob: Send Amounts
+    note right: Btc & Xmr Amounts
+end
```

Comment:
I'm a bit confused on the first step: 
Bob is the maker (wanting BTC) and Alice the taker (wanting XMR).
Bob comes up with the rate. So the first request should be from Alice to Bob : `Get Rate ` or similar.

Reply:
This is a direct translation of the code. I intend to remove this part as indeed, it is not helpful.

Reply:
My initial comment was bollocks: Alice has XMR and wants BTC and Bob has BTC and has XMR.